packages/frontend/src/App.svelte

```diff
@@ -26,6 +33,9 @@ onMount(() => {
       <Route path="/" breadcrumb="Bootable Containers">
         <Homepage />
       </Route>
+      <Route path="/build2/:name/:tag" breadcrumb="Build" let:meta>
```

Comment:
What we could do instead is modify it to `<Route path="/build/:name/:tag" breadcrumb="Build" let:meta>` / have the same route.

And then within Build.svelte if name / tag is blank, do the normal current functionality.